src/quorum/testdata/joint_group_commit.txt

```diff
@@ -0,0 +1,76 @@
+# the same result of joint single group commit
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(1,1,1)
+----
+100
+
+# min(quorum_commit_index = 100, first index that appears in second group = 99)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(1,1,2)
+----
+99
+
+# min(quorum_commit_index = 100, first index that appears in second group = 101)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(2,1,1)
+----
+100
+
+# minimum index = 99
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(_,1,1)
+----
+99
+
+# min(quorum_commit_index = 100, first index that appears in second group = 99)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(_,1,2)
+----
+99
+
+# minimum index = 98
+group_committed cfg=(1,2,3,4,5) cfgj=zero idx=(100,101,99,102,98) gid=(_,_,_,_,1)
+----
+98
+
+# cfg 1 = min(quorum_commit_index = 100, first index that appears in second group = 99) 
+# cfg 2 = min(quorum_commit_index = 100, first index that appears in second group = 1) 
+group_committed cfg=(1,2,3,4) cfgj=(3,4,5,6) idx=(101,99,100,102,103,1) gid=(1,_,1,1,_,2)
+----
+1
+
+# cfg 1 = 100
+# cfg 2 = 100
```

Comment:
It will be good to have comment with unified formatted for configurations that have the second group like:
`# minimum index = xx`
or
`# cfg x = min(quorum_commit_index = xx, first index that appears in second group = xx)`